test/definitions/checks/check_ipv6_disable_test.rb

```diff
@@ -0,0 +1,31 @@
+require 'test_helper'
+
+describe Checks::CheckIpv6Disable do
+  include DefinitionsTestHelper
+
+  subject { Checks::CheckIpv6Disable.new }
+
+  context 'throw an error message when ipv6.disable=1 is set' do
+    before do
+      File.expects(:read).with('/proc/cmdline').returns('ipv6.disable=1')
+    end
+
+    it 'system is self registered' do
```

Comment:
I bet you can come up with a better description ;)

Reply:
How much you willing to bet?

Reply:
A dime?